src/cmd/root.go

```diff
@@ -48,6 +48,7 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVar(&Config.InfraRepoSuffix, "infra-repo-suffix", "-ci", "Suffix for infrastructure git repository")
 	rootCmd.PersistentFlags().StringVar(&Config.ImageTag, "tag", "", "Commit-Hash/Image-Tag for the deployment change")
+	rootCmd.PersistentFlags().StringVar(&Config.ImageTagFilename, "image-tag-file-name", "values.yaml", "Name of the file in which the image tag can be found")
```

Comment:
image-tag-file-name find ich einen weirdly specific name. in dem values.yaml bzw. auch in der kubevela definition steht vermutlich mehr drinnen, als nur der image-tag. 
ein anderer name wäre meiner meinung nach besser. vielleicht in die richtung "deployment-config-file" oder "configuration-file" oder "app-config-file" irgendwie in die richtung